gsadjust/pyqt_models.py

```diff
@@ -729,15 +729,11 @@ def gravnet_inversion(self):
                 drift_term = '-T' + str(unique_ls[0])
 
         # Remove old gravnet files
-        try:
-            os.remove(self.name + '.gra')
```

Comment:
Exceptions need to be caught for each operation individually, otherwise after the first failure none of the following files will attempt to delete, e.g. if `.err` is missing, none of the other files will be deleted.